### DISCUSSION TITLE
How to do website statistics?

### DISCUSSION THREAD

**Discussion by GaiheiluKamei** [bridgetownrb/bridgetown#822] [Q&A] [answered]
I want to get traffic statistics for my blog site, like those fancy curve charts, how do I do that? Any good ideas on using plugins or third-party websites?

**Comment by sandstrom** [comment, marked as answer]
It's totally outside the scope of Bridgetown.

But you can use e.g. https://usefathom.com/, https://plausible.io/ or Google analytics.

**Reply by GaiheiluKamei** [reply]
Thank you so much and sorry for didn't reply in time.

I just don't have any mind about how to do this, because the course I learned said that I should have a "view" column in the database, and update it every time the "show" action gets triggered, which will be a pressure to the database.
